app/src/main/res/layout/activity_option2.xml

```diff
@@ -42,4 +42,28 @@
         android:textAppearance="@style/text_title"
         android:background="@color/colorAccent"/>
 
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <Button
+            android:id="@+id/btn_next0"
+            android:layout_width="wrap_content"
+            android:layout_height="@dimen/touch_target_size"
+            android:layout_marginBottom="@dimen/small_spacing"
+            android:onClick="process"
```

Comment:
Just found this after recheck. I think you may need to think about this since you added "setOnClickListener" in Java file. Do you still need to put this "onClick" attribute?